apps/learn.svelte.dev/vercel.json

```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "https://openapi.vercel.sh/vercel.json",
+	"redirects": [
+		{
+			"source": "/(.*)",
+			"destination": "https://svelte.dev/$1",
+			"permanent": true
```

Comment:
might want to skip `permanent` on whatever fallback we have for this one so that if there's some URL we missed we have an opportunity to fix it